tools/militarytools/esri/toolboxes/scripts/ConversionUtilities.py

```diff
@@ -803,15 +803,45 @@ def tableToPoint(inputTable,
             scratch = env.scratchWorkspace
             
         inputSpatialReference = _checkSpatialRef(inputSpatialReference)
-        
-        arcpy.ConvertCoordinateNotation_management(inputTable,
+        if inputSpatialReference != arcpy.SpatialReference(4326): #default is GCS_WGS_1984 - if the CS is different 
```

Comment:
@BobBooth - I don't know if you can compare these objects directly `*`, it might be safer to compare the factoryCode or Name 

(`*` without knowing if this object implements the `_eq_` operator)